GVFS/GVFS.Common/PlaceholderListDatabase.cs

```diff
@@ -73,6 +69,15 @@ public static bool TryCreate(ITracer tracer, string dataFilePath, PhysicalFileSy
             return true;
         }
 
+        /// <summary>
+        /// The Count is "estimated" because it's simply (# adds - # deletes).  There is nothing to prevent
+        /// multiple adds or deletes of the same path from being double counted
+        /// </summary>
+        public int Count()

```

Comment:
Maybe `EstimatedCount`?

Reply:
This is part of the interface so it is just `Count()` because with SQLite it is not an estimate since it is running a SQL query to get the count.  This class will basically be deprecated with these changes and shouldn't be used except for in the disk layout upgrade.  I have even thought about renaming it to `DeprecatedPlaceholderListDatabase` to make that obvious.  What do you think?

Reply:
Maybe add the `[Obsolete]` attribute? Although that would introduce a warning at each use-site (which could be `#pramga disabled` in the layout upgrade code..).